Pages/badge.html

```diff
@@ -224,19 +225,19 @@ <h3 class="my-2">Examples</h3>
             <div class="example-box my-4">
                 <button class="badge-btn s-8">
                     <a href="#">
-                        <img class="badge-img img-s8" src="https://i.pravatar.cc/300?img=57" alt = "image" />
+                        <img class="badge-img img-s8" src="https://i.pravatar.cc/300?img=57" alt="image" />
                         <span class="badge badge-red"></span>
                     </a>
                 </button>
                 <button class="badge-btn s-8">
                     <a href="#">
-                        <img class="badge-img img-s8" src="https://i.pravatar.cc/300?img=61" alt = "image" />
+                        <img class="badge-img img-s8" src="https://i.pravatar.cc/300?img=61" alt="image" />
                         <span class="badge badge-red"></span>
                     </a>
                 </button>
                 <button class="badge-btn s-8">
                     <a href="#">
-                        <img class="badge-img img-s8" src="https://i.pravatar.cc/300?img=" alt = "image" />
+                        <img class="badge-img img-s8" src="https://i.pravatar.cc/300?img=" alt="image" />
```

Comment:
you can be more specific while naming class, instead of img-s8 you can name something that is more readable.